python/nano/test/onnx/pytorch/test_onnx.py

```diff
@@ -177,6 +177,64 @@ def test_onnx_context_manager(self):
             assert torch.get_num_threads() == 1
             output = onnx_model(x)
 
+    def test_onnx_default_values(self):
+        # default bool values
+        class Net(nn.Module):
+            def __init__(self):
+                super().__init__()
+            def forward(self, x, a=True, b=False):
+                if a:
+                    return x+1
+                if b:
+                    return x-1
+                return x
+
+        model = Net()
+
+        data = torch.rand(1,3,1,1)
+        result_true = model(data)
+        # sample with only required parameters (in a tuple)
+        accmodel = InferenceOptimizer.trace(model,
+                                            accelerator="onnxruntime",
+                                            input_sample=(torch.rand(2,3,1,1),))
+        result_m = accmodel(data)
+        assert torch.equal(result_true, result_m)
+
+        # sample with only required parameters
+        accmodel = InferenceOptimizer.trace(model,
+                                            accelerator="onnxruntime",
+                                            input_sample=torch.rand(2,3,1,1))
+        result_m = accmodel(data)
+        assert torch.equal(result_true, result_m)
+
+        data = torch.rand(1,3,1,1)
+        result_true = model(data, False, True)
+        # sample with only required parameters
+        accmodel = InferenceOptimizer.trace(model,
+                                            accelerator="onnxruntime",
+                                            input_sample=(torch.rand(2,3,1,1),False,True))
+        result_m = accmodel(data)
+        assert torch.equal(result_true, result_m)
+
+        # typehint model
+        class Net(nn.Module):
+            def __init__(self):
+                super().__init__()
+            def forward(self, x: torch.Tensor, y: int):
+                return x+y
+
+        model = Net()
+
+        x = torch.rand(1,3,1,1)
+        y = 3
+        result_true = model(x, y)
+        # sample with only required parameters (in a tuple)
+        accmodel = InferenceOptimizer.trace(model,
+                                            accelerator="onnxruntime",
+                                            input_sample=(torch.rand(2,3,1,1),3))
```

Comment:
in this case, can we only pass in torch.rand(2,3,1,1)?

Reply:
change a little bit of the uts. Done